setup.py

```diff
@@ -67,18 +69,17 @@ def get_scm_config():
                    'hpy/debug/src/dhqueue.c',
                    'hpy/debug/src/_debugmod.c',
                    'hpy/debug/src/autogen_debug_wrappers.c',
-                  ],
+                   ],
```

Comment:
I'm happy to merge PRs to improve readability, but I prefer if we stick to existing conventions. So far all closed brackets are indented at the same level, this breaks the rule

Reply:
That's all good!